### PR TITLE
SWQ-346 Added script that sets up a freshly installed Ubuntu PC

### PR DESCRIPTION
**Pull request by HEA-bogo**
Just a script that runs all the steps to setup on a fresh Ubuntu installation.

### CODE REVIEW

File: devscript.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+fi
+
+sudo apt install -y python3-pip git
+
+sudo pip3 install --upgrade pip
+sudo pip3 install pipenv
+
+mkdir repos
+cd repos
+git clone https://github.com/bang-olufsen/autotest
```
Comment (jeppefrandsen) [inline devscript.sh]: I think the script should be placed in autotest and not here. All people should know how to clone a repository

Comment (jeppefrandsen) [reply devscript.sh]: Please also note that this it a public repo

Comment (HEA-bogo) [reply devscript.sh]: :+1:  
Will move it to Autotest.
The reason I clone here, is just so you don't have to do it manually :smile:

Comment (jeppefrandsen) [reply devscript.sh]: Great - yes, agree ;)

**Pull request closed without merging**